src/cdn.rs

```diff
@@ -123,7 +123,17 @@ pub(crate) fn invalidate_crate(config: &Config, cdn: &CdnBackend, name: &str) ->
             distribution_id,
             &[&format!("/{}*", name), &format!("/crate/{}*", name)],
         )
-        .context("error creating CDN invalidation")?;
+        .context("error creating web CDN invalidation")?;
+    }
+    if let Some(distribution_id) = config.cloudfront_distribution_id_static.as_ref() {
+        cdn.create_invalidation(
+            distribution_id,
+            &[
+                &format!("/rustdoc/{}*", name),
+                &format!("/sources/{}*", name),
```

Comment:
Since we're not tagging these public `/sources` paths shouldn't need invalidating yet?

Reply:
In my mind I added it just for the sake of completness, though I see that it's rather unlikely that we offer source download :)

Reply:
( from a AWS limit standpoint it doesn't make a difference with this amount of paths)